tests/test_measure.py

```diff
@@ -44,10 +44,10 @@ def get_meas_generator(meas_params, multiprocessing=False, cpus=1):
 
 def compare_sep():
     """Test detection with sep"""
-    meas_param = btk.measure.SepParams()
-    meas_generator = get_meas_generator(meas_param)
-    _, deb, _ = next(meas_generator)
-    detected_centers = deb[0]["peaks"]
+    meas_generator = get_meas_generator(btk.measure.sep_measure)
+    _, results = next(meas_generator)
+    dx, dy = (results[0]["catalog"]["dx"].item(), results[0]["catalog"]["dy"].item())
```

Comment:
IMO, one should be able to extract two result values with much straightforward code.
This would mean refactoring a bit the output of the measurement methods.
Probably for a next PR.